cue/ui/gen.cue

```diff
@@ -69,6 +69,7 @@ HideSeriesConfig: {
 	tooltip: bool
 	legend:  bool
 	viz:   bool
+	graph?: bool
```

Comment:
some of the timeserie panel type still get this field.

Reply:
> some of the timeserie panel type still get this field.

Where?  they should not

Reply:
So we should either change that devenv dashboard JSON to use `viz`, or keep this change and replicate it to the cue file under packages/ from which it derives

Reply:
those are at the following docs. I was baffled, would correct the JSON instead since the frontend changed already.
devenv/dev-dashboards/datasource-testdata/new_features_in_v8.json
devenv/dev-dashboards/panel-graph/graph-ng-nulls.json
devenv/dev-dashboards/panel-barchart/barchart-autosizing.json
devenv/dev-dashboards/datasource-elasticsearch/elasticsearch_compare.json

Reply:
> > some of the timeserie panel type still get this field.
> 
> Where? they should not

those are at the following docs. I was baffled, would correct the JSON instead.
devenv/dev-dashboards/datasource-testdata/new_features_in_v8.json
devenv/dev-dashboards/panel-graph/graph-ng-nulls.json
devenv/dev-dashboards/panel-barchart/barchart-autosizing.json
devenv/dev-dashboards/datasource-elasticsearch/elasticsearch_compare.json

Reply:
my concern is that I have the impression front end made a "backward compatible change", in the file, for example, barchart-autosizing.json, both fields are present together.